tensorboard_plugin_wit/wit_plugin_loader.py

```diff
@@ -38,6 +39,14 @@ def load(self, context):
             import tensorflow
         except ImportError:
             return
+
+        # If TB version is before 2.2.0, then do not load the WIT plugin
+        # as it is already included directly in TensorBoard.
+        version_list = tensorboard.__version__.split(".")
```

Comment:
Note: `tensorboard.__version__` was [added in TensorBoard 1.14.0][1]. If
you want to be back-compatible with older versions, you can use:

```python
import tensorboard.version

tb_version = tensorboard.version.VERSION
```

[1]: https://github.com/tensorflow/tensorboard/pull/2026

Reply:
don't think we'll need backwards compat with 1.14, as colab falls back to 1.15 with the tensorflow 1.x magic and anyone installing 1.14 from scratch won't get this pip package